src/app/shared/pagination/pagination.component.ts

```diff
@@ -105,23 +103,18 @@ export class PaginationComponent implements OnDestroy, OnInit {
    * In other words, changing pagination won't add or update the url parameters on the current page, and the url
    * parameters won't affect the pagination of this component
    */
-  @Input() public disableRouteParameterUpdate = false;
+  @Input() public retainScrollPosition = false;
```

Comment:
has this param been only renamed or has it also a different meaning ?

Reply:
the previous param was used to decide if update or not the route with pagination params. If I understand well this params has a different scope. In my opinion the old functionality should be preserved as well, because not always the pagination component should affect the route. The pagination used within a modal is a good example, because it is unnecessary to update the route

Reply:
It is not strictly necessary for a modal I agree, but is it a problem? It could be seen as an advantage to be able to share or bookmark the URL of a specific page on a modal, even if you still have to open the modal manually to see it. By storing pagination state in the URL all the time, we're also getting rid of complexities and edge cases you would get by storing it in the elsewhere only if disableRouteParameterUpdate is true.